suite-native/module-accounts-import/src/screens/AccountImportLoadingScreen.tsx

```diff
@@ -85,18 +86,15 @@ export const AccountImportLoadingScreen = ({
             if (!ignore) {
                 if (fetchedAccountInfo?.success) {
                     if (networkSymbol === 'eth') {
```

Comment:
Do we want to support even 'matic' here?

Reply:
We don't have support for matic yet. There will be many changes needed later. I would rather do not create any code that is related to other tokens then ETH ERC20. It could make this PR very large and even introduce some unwanted bugs as side effect. What do you think?

Reply:
I agree, I just wanted to know your thoughts in light of previous `isEthereumBasedNetwork` appearance in code.